pkg/platypus/client.go

```diff
@@ -207,19 +207,24 @@ func (p *PlaidClient) UpdateItem(ctx context.Context) (LinkToken, error) {
 
 	log := p.getLog(span)
 
-	log.Trace("creating link token for update")
-
-	redirectUri := fmt.Sprintf("https://%s/plaid/oauth-return", p.config.OAuthDomain)
+	var redirectUri *string
+	if p.config.OAuthDomain != "" {
+		redirectUri = myownsanity.StringP(fmt.Sprintf("https://%s/plaid/oauth-return", p.config.OAuthDomain))
+		log = log.WithField("redirectUri", *redirectUri)
+	}
```

Comment:
In the future we should figure out a way to allow for https local development domains. This way we can support OAuth more easily for local dev as Plaid now requires https redirect URIs.